<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>906</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <addaction name="actionBlur"/>
    <addaction name="actionGaussian_Blur"/>
    <addaction name="actionBox_Filter"/>
    <addaction name="actionMedian_Filter"/>
    <addaction name="actionBilateral_Filter"/>
    <addaction name="actionMedian_threshold"/>
   </widget>
   <widget class="QMenu" name="menuOrder">
    <property name="title">
     <string>Order</string>
    </property>
    <addaction name="actionHistogram"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuImage"/>
   <addaction name="menuOrder"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionBlur">
   <property name="text">
    <string>Blur</string>
   </property>
  </action>
  <action name="actionBox_Filter">
   <property name="text">
    <string>Box Filter</string>
   </property>
  </action>
  <action name="actionMedian_Filter">
   <property name="text">
    <string>Median Filter</string>
   </property>
  </action>
  <action name="actionGaussian_Blur">
   <property name="text">
    <string>Gaussian Blur</string>
   </property>
  </action>
  <action name="actionBilateral_Filter">
   <property name="text">
    <string>Bilateral Filter</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="text">
    <string>Histogram</string>
   </property>
  </action>
  <action name="actionMedian_threshold">
   <property name="text">
    <string>Median threshold</string>
   </property>
  </action>
  <action name="actionErlang">
   <property name="text">
    <string>Erlang</string>
   </property>
  </action>
  <action name="actionRayleigh">
   <property name="text">
    <string>Rayleigh</string>
   </property>
  </action>
  <action name="actionUniform">
   <property name="text">
    <string>Uniform</string>
   </property>
  </action>
  <action name="actionGamma">
   <property name="text">
    <string>Gamma</string>
   </property>
  </action>
  <action name="actionExponal">
   <property name="text">
    <string>Expomential</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
